.github/workflows/deploy.yml

```diff
@@ -17,15 +17,17 @@ jobs:
           node-version: 12.x
 
       - name: Cache Node.js modules
-        uses: actions/cache@v1
+        uses: actions/cache@v2
+        id: cache
         with:
-          path: ~/.npm
+          path: ./node_modules
           key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
             ${{ runner.OS }}-node-
             ${{ runner.OS }}-
 
       - name: Install npm dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
         run: npm i
```

Comment:
Try `npm ci` here, with no skip, and cached `~/.npm`

Reply:
Yep we can change to `npm ci` but it is not much fast but it is a better way to do it 

Deploy#22, Deploy#21 is where cached `.npm` is used 
![image](https://user-images.githubusercontent.com/44710980/84681139-55ee6d00-aef9-11ea-90d6-96356106ccef.png)